lib/template_handler.js

```diff
@@ -19,7 +19,7 @@ module.exports = {
 			return false;
 		}
 
-		var path_portions = template_path.split(Path.sep || "/");
+		var path_portions = template_path.split("/");
```

Comment:
so in template_handler.js `isSection: function(template_path){}`
the `template_path` variable contains paths formatted like `/images\banner`.